person.rb

```diff
@@ -0,0 +1,36 @@
+require './nameable'
+require './decorators'
```

Comment:
- [ ] Good job so far, :clap: However, Please ensure that you remove the `./` and use `nameable` to target the project directory. 
- [ ] Please ensure that you remove the `./` and use `decorators` to target the project directory. 
- [ ] Kindly use `require_relative` as the most recommended way. :slightly_smiling_face: 
- Typical Example: :arrow_down: 
`require_relative 'nameable'`
`require_relative 'decorators'`